website/src/views/JoinClass/JoinClass.test.jsx

```diff
@@ -2,15 +2,31 @@ import React from 'react';
 import renderer from 'react-test-renderer';
 import {
   configure,
+  shallow,
 } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import { JoinClass } from '..';
 
 configure({ adapter: new Adapter() });
 
 
+/* Snapshot */
 describe('JoinClass', () => {
   it('Render JoinClass without any eror', () => {
     expect(renderer.create(<JoinClass />).toJSON()).toMatchSnapshot();
   });
 });
+
+/* Enzyme */
+const wrapper = shallow(<JoinClass />);
```

Comment:
The more that i think about the less these tests make sense. It is essentially testing that the page is organized in the way we designed it but that is accomplished by the snapshot test. It might be better to write a test to see if the functionality works. So for example check that the search works and suggestions pop up. That your able to click on one of them and it gets saved to the state.

Reply:
We can discuss it tonight.

Reply:
Continue with this topic, there are the simulate method for testing the onClick and setState method in the enzyme to help u to test those what Mohammed  said